OIPA/iati/parser/tests/test_2_03_parser.py

```diff
@@ -5655,3 +5655,161 @@ def test_activity_fss(self):
         except ParserError as inst:
             self.assertEqual(inst.field, 'Fss')
             self.assertEqual(inst.message, 'must occur no more than once.')
+
+
+class ActivityFssForecastTestCase(TestCase):
+    '''
+       2.03: Added new (optional) <forecast> element for <fss>
+       element.
+    '''
+    def setUp(self):
+        # 'Main' XML file for instantiating parser:
+        xml_file_attrs = {
+            "generated-datetime": datetime.datetime.now().isoformat(),
+            "version": '2.03',
+        }
+        self.iati_203_XML_file = E("iati-activities", **xml_file_attrs)
+
+        dummy_source = synchroniser_factory.DatasetFactory.create()
+
+        self.parser_203 = ParseManager(
+            dataset=dummy_source,
+            root=self.iati_203_XML_file,
+        ).get_parser()
+
+        # Related objects:
+        # create dummy object
+        self.fss = iati_factory.FssFactory.create()
+        self.currency = iati_factory.CurrencyFactory.create()
+        self.parser_203.register_model('Fss', self.fss)
+        self.parser_203.register_model('Currency', self.currency)
+
+    def test_activity_fss_forecast(self):
+        """
+        Test if attributes in <fss/forecast> XML
```

Comment:
Also just a small thing, but I don't think such XML element exists in the standard. Docstrings in python have a purpose of 1) running tests, or 2) **documenting code**. So for the latter one - they should be precise, because it's documentation.